raiden/network/transport/matrix/client.py

```diff
@@ -227,7 +227,7 @@ def create_room(
     def get_presence(self, user_id: str) -> Dict[str, Any]:
         return self._send("GET", f"/presence/{quote(user_id)}/status")
 
-    def invite(self, room_id, offer):
+    def invite(self, room_id: str, offer: Dict[str, str]) -> None:
```

Comment:
Can you type this with `RoomID`?

Reply:
in the whole transport and matrix client code base it is named `room_id`. I'd suggest we leave it consistent here.

Reply:
I meant the type itself: `room_id: RoomID`

Reply:
you are perfectly right on this